src/index.js

```diff
@@ -14,11 +42,8 @@ export class Entity extends React.Component {
 
   attachEvents = el => {
     if (!el) { return; }
-    Object.keys(this.props.events).forEach(eventName => {
-      el.addEventListener(eventName, event => {
-        this.props.events[eventName](event);
-      });
-    });
+    attachAframeEventsToElement(el, this.props.events);
```

Comment:
Can name it just `attachEventsToElement`

We can build the object all at once and then call `attachEventsToElement` once